### ISSUE TITLE
Document standard practice for linking ontology term to ticket discussions

### ISSUE THREAD

**Issue by mellybelly** [OBOFoundry/OBOFoundry.github.io#1097]
It would be great to have a standardized practice for this. I know various ontologies do this to some extent but we could have a best practice for it (if we don't already).

**Comment by matentzn** [comment]
Yeah, good idea. I would personally prefer axiom annotations over term annotations (because this hides them a bit better), but I can see that probably the discussions won't be limited to single aspects of the term. 

So here my two suggestions:

Given a term T:

- [ ] Option A: `T rdfs:seeAlso "https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1097"`
- [ ] Option B: `T rdf:label "T LABEL" [rdfs:seeAlso "https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1097]`

Option A is more visible, and people can quickly find related discussions, but also clutters the ontology and user facing apps may have to do some filtering if they dont want to show stuff that may be only relevant to the curators. Option B is nice and hidden, and since every class has a label (or should have) and rdf:label is fairly common place, maybe it could go there as well. 

Any other suggestions?

**Comment by balhoff** [comment]
This just came up on the GO editors call yesterday. @ukemi @pgaudet do you have any comments?

I don't really like hiding it on the label axiom. That's very confusing. :-)

**Comment by matentzn** [comment]
Yeah on second thought, I think I would agree. Maybe it should be a more first class citizen, sitting directly on the term.

**Comment by zhengj2007** [comment]
A specific annotation property has been added in IAO ontology-metadata to indicate the tracker associated with a term:
http://purl.obolibrary.org/obo/IAO_0000233 term tracker item
Definition: An IRI or similar locator for a request or discussion of an ontology term.

**Comment by balhoff** [comment]
Thanks @zhengj2007, that looks perfect. I think we should take care that these links are IRI values, and not strings.

**Comment by pgaudet** [comment]
Hi @balhoff 

I have added a link to the GH ticket in the 'comments' field but it would be better if there was either a specific field for this information, or if we'd tag it somehow. Note also that when there are edits, multiple tickets may be relevant. 

Thanks, Pascale

**Comment by jonquet** [comment]
Although I like IAO, why not use a property from a W3C Recommendation such as PROV-O. For instance, 
https://www.w3.org/TR/prov-o/#hadPrimarySource
or 
https://www.w3.org/TR/prov-o/#wasInformedBy

PROV-O is explicitly made for storing provenance information. 
(it might also be the moment to suggest to adopt a couple of others PROV-O properties for other thing e.g., https://www.w3.org/TR/prov-o/#wasAttributedTo for attribution of authorship rather than hasDBXrefs)

**Comment by pgaudet** [comment]
Isn't this a little different from provenance ? I would think provenance is the paper that shows the existence of the term, and helps define it. 

The ticket is the request that led to the creation/edition/obsoletion of the term. These seem like different things to me. 

Thanks, Pascale

**Comment by jonquet** [comment]
Provenance is all about that. Check out PROV-O and PAV that are standard vocabs to store provenance information. You will see the amount of different things these vocabs help to represent.

The question is which semantic web property shall be used to represent a specific information about a term. Obsoletion for instance is usually stored with the owl:deprecated property. Creation by the dc:creator or the prov:wasAttributedTo properties.

Check out : https://link.springer.com/article/10.1007/s13740-018-0091-5 
We have reviewed 346 properties from 22 vocabularies and "merged" them into a model of 127... all of these to describe ontologies.

**Comment by cmungall** [comment]
@jonquet - thanks for all your detailed work here! While I agree that prov:wasInformedBy is not wrong -- the ticket generally does inform the content of the class. However, there are advantages to using a more precise property, it makes it easier to fetch all gh tickets without doing string matching on the value

perhaps the iao prop could be a subProp of prov:wasInformedBy?

**Comment by cmungall** [comment]
I do think there are other cases where IAO has minted a property where a more widely used semweb property exists. We should consider whether to obsolete the IAO term in these cases.

**Comment by nlharris** [comment]
Is there still work to be done on this?

**Comment by nataled** [comment]
The participants of the OFOC call 2025-08-19 has accepted the EWG proposal that the practice discussed here be accepted and incorporated into a document of miscellaneous guidelines that the EWG is putting together. Briefly, that document will contain wording to the effect of:

If there is an issue tracker item for a particular term or set of terms (for example, a new term request or a term discussion), then the relevant terms SHOULD be linked to the appropriate tracker issue. Instead of using a free text comment, the link between the term and the tracker item SHOULD be the annotation property 'term tracker item' (IAO:0000233). The range for this property is an IRI,
without additional text or comment, that MUST point to an issue tracker item for the annotated term.
